timemachine/fe/free_energy.py

```diff
@@ -601,22 +633,31 @@ def run_production_local_steps(n_steps: int) -> Tuple[NDArray, NDArray, NDArray]
             coords.append(x_t)
             boxes.append(box_t)
 
-        assert np.all(coords[-1][-1] == ctxt.get_x_t())
```

Comment:
Is there a real performance impact to removing these assertions? (or are they replicated elsewhere that I'm missing?) They seem like a nice sanity check, but probably not necessary if this code is sufficiently tested.

Reply:
Don't think so, but would rather not do work we don't have to. I am inclined to say it is sufficiently tested since I changed this behavior in https://github.com/proteneer/timemachine/pull/724 if we broadly feel that it isn't, I can revert these changes in this PR. Then follow up with a PR that removes them and adds more test.